notebooks/Analyze_Text.ipynb

```diff
@@ -0,0 +1,2895 @@
+{
```

Comment:
<p>Might be good to make it clear that the first path is raw data + python and only to show how messy it is compared to the neat Pandas work following. It's a lot for the user to look at and they might not know we are going to show a much better version.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Good point. I've expanded on the introduction here with a description of the two implementations we're about to do and some warnings that the non-Pandas version of the syntax analysis output is a bit daunting.</p>